src/librustc_mir/interpret/terminator.rs

```diff
@@ -463,7 +451,7 @@ impl<'mir, 'tcx, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
 
     /// Evaluate a const function where all arguments (if any) are zero-sized types.
     /// The evaluation is memoized thanks to the query system.
-    fn eval_const_fn_call(
+    pub (crate) fn eval_const_fn_call(
```

Comment:
```suggestion
    // FIXME: Consider moving this to `const_eval.rs`.
    pub (crate) fn eval_const_fn_call(
```
@oli-obk could you add this soon-ish? I think this PR has to land to fix the Miri toolstate (as calling `is_const_fn_raw` on closures does not work any more since https://github.com/rust-lang/rust/pull/66275)